### PR TITLE
Use inputs and outputs nomenclature

### PR DESCRIPTION
**Pull request by argjv**
Destination and sender is not a good name. Inputs and outputs are easier to understand. It also makes it easier to consume by library users.

### CODE REVIEW

File: src/coin/baseCoin/iface.ts [hunk 1]
```diff
@@ -24,6 +24,6 @@ export type ExtendedKeys = {
   xpub: string;
 }
 
-export interface Destination extends BaseAddress {
+export interface Entry extends BaseAddress {
```
Comment (CapnMigraine) [inline src/coin/baseCoin/iface.ts]: nit: unrelated to this change, but can we make `address` in `BaseAddress` a string?

File: src/coin/trx/transaction.ts [hunk 2]
```diff
@@ -48,33 +48,37 @@ export class Transaction extends BaseTransaction {
...
+        const value = new BigNumber((rawData.contract[0] as TransferContract).parameter.value.amount);
+        output = {
           address: (rawData.contract[0] as TransferContract).parameter.value.to_address,
-          value: new BigNumber((rawData.contract[0] as TransferContract).parameter.value.amount),
+          value,
         };
-        senderAddress = {
+        input = {
           address: (rawData.contract[0] as TransferContract).parameter.value.owner_address,
+          value,
```
Comment (CapnMigraine) [inline src/coin/trx/transaction.ts]: shouldn't we negate input values as we do in entries created by indexer and platform?

File: src/coin/trx/transaction.ts [hunk 3]
```diff
@@ -130,6 +134,16 @@ export class Transaction extends BaseTransaction {
     }
   }
 
+  get signature(): string[] {
```
Comment (CapnMigraine) [inline src/coin/trx/transaction.ts]: nit: add docstring

File: src/coin/trx/transaction.ts (hunk 2 above)
Comment (argjv) [reply src/coin/trx/transaction.ts]: Good point, but these are inputs and outputs that go into separate fields. If they were mixed up, then yeah, we should negate the values.

**Pull request merged**